README.md

```diff
@@ -114,19 +114,13 @@ These options can be set to `false` or `'transform'`. When using `'transform'`,
   // → '(.)\1'
   ```
 
-#### Experimental regular expression features
-
-These options can be set to `false`, `'parse'` and `'transform'`. When using `'transform'`, the corresponding features are compiled to older syntax that can run in older browsers. When using `'parse'`, they are parsed and left as-is in the output pattern. When using `false` (the default), they result in a syntax error if used.
-
-Once these features become stable (when the proposals are accepted as part of ECMAScript), they will be parsed by default and thus `'parse'` will behave like `false`.
-
 - `unicodeSetsFlag` - [The `v` (`unicodeSets`) flag](https://github.com/tc39/proposal-regexp-set-notation)
```

Comment:
The GH diff is a bit confusing: Here the `unicodeSetsFlag` section is moved from experimental to stable.